roles/ceph-common/tasks/misc/system_tuning.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: disable osd directory parsing by updatedb
+  command: updatedb -e /var/lib/ceph
+  changed_when: false
+  ignore_errors: true
+
+- name: disable transparent hugepage
+  command: "echo never > /sys/kernel/mm/transparent_hugepage/enabled"
+  changed_when: false
+  ignore_errors: true
+  when: disable_transparent_hugepage
+
+- name: disable swap
+  command: swapoff -a
```

Comment:
This can be disruptive on a running system which is using swap, but already handled in the common role.